daml/Tests/Bond.daml

```diff
@@ -29,11 +30,14 @@ testBond = script do
   lifecycleReqCid <- submit bob.customer $ exerciseCmd bob.custodyServiceCid RequestLifecycle with
     assetDepositCid = bondCid
     choice = bobsDesc.claims
-  bondCid :: _ <- submit providers.bank $ exerciseCmd custodyServiceCid $ Lifecycle with
+  (bondCid, [settlementInstructionCid]) <- submit providers.bank $ exerciseCmd custodyServiceCid $ Lifecycle with
     safekeepingDepositCid = corpActDepositCid
     lifecycleRequestCid = lifecycleReqCid
     fixings = Map.fromList []
 
+  submitMulti [alice, bob.customer, providers.bank] [] $
```

Comment:
At present don't think this is supported in Daml Hub :( @brian-weir cc

Reply:
You are right (we had the same problem somewhere).
It is OK to have this in the tests, not? (They are just tests, not run on Daml Hub.)
In "production" (on Hub) one can create a couple of contracts to collect signatures of those three parties (seller, buyer, provider I think).

Reply:
@brian-weir cc